README.md

```diff
@@ -44,7 +44,7 @@ It's a one key solution to all your tmux session needs.
 
 ### ⚡ Requirements
 
-- tmux, fzf, zoxide (required)
+- tmux, fdfind, fzf, zoxide (required)
```

Comment:
I just realized that fd was not mentioned here earlier, lets change this to fd please